newsfragments/1573.bugfix.rst

```diff
@@ -0,0 +1 @@
+Added a helpful error message if an async function is passed to `trio.to_thread_run_sync`.
```

Comment:
You have a typo here which fails the docs build: the exported name is `trio.to_thread.run_sync`